### PR TITLE
[ci][nix][fastlane] add building unsigned Expo Client

### PR DESCRIPTION
**Pull request by dsokal**
We'd like to enable users to build their our Expo Client with Turtle. This is the first thing we need - the PR adds a CI job which builds an unsinged Expo Client "shell app".

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -20,6 +21,7 @@ commands:
           # xz: circleci checkout steps
           command: |
             nix-env -iA \
+              nixpkgs.awscli \
```
Comment (nicknovitski) [inline .circleci/config.yml]: This shouldn't be necessary, there's already a binstub for the aws cli.
```suggestion
```

File: .circleci/config.yml [hunk 2]
```diff
@@ -9,6 +9,7 @@ commands:
     steps:
       - run:
           name: Install common packages
+          # awscli: used to upload Expo Client shell app to S3
```
Comment (nicknovitski) [inline .circleci/config.yml]: This shouldn't be necessary, there's already a binstub for the aws cli.
```suggestion
```

File: nix/default.nix [hunk 3]
```diff
@@ -15,5 +15,7 @@ in
       allowUnfree = true;
       android_sdk.accept_license = true;
     };
-    overlays = [ (import ./all-packages.nix) ];
+    overlays = [
+      (import ./all-packages.nix)
+    ];
```
Comment (nicknovitski) [inline nix/default.nix]: Was this change intentional?  I don't object to it, but I don't see the need either.

**Review by nicknovitski — APPROVED** [review]

File: .circleci/config.yml (hunk 1 above)
Comment (wkozyra95) [reply .circleci/config.yml]: There is in the `expo/universe` repo but not here, unless I missed sth.

**Review by esamelson — APPROVED** [review]

File: .circleci/config.yml (hunk 1 above)
Comment (nicknovitski) [reply .circleci/config.yml]: https://github.com/expo/expo/blob/master/bin/aws

Comment (wkozyra95) [reply .circleci/config.yml]: This is PR for sdk-32 branch (I'm not sure why) and aws cli is not available there. Dominik will be back on Monday, so I'll probably leave this to him.
